README.md

```diff
@@ -52,75 +52,37 @@ Create a file named `your-username.json` in the directory `public/data` with the
 
 Optional fields: `links` and `milestones`
 
+Looking for inspiration? You can view the following profiles for an example:
+
+- [Eddie Jaoude](https://github.com/EddieHubCommunity/LinkFree/blob/main/public/data/eddiejaoude.json)
+- [nhcarrigan](https://github.com/EddieHubCommunity/LinkFree/blob/main/public/data/nhcarrigan.json)
```

Comment:
```suggestion
- [Nicholas Carrigan](https://github.com/EddieHubCommunity/LinkFree/blob/main/public/data/nhcarrigan.json)
```

Reply:
I have myself listed as `nhcarrigan` on the site - this should reflect that. 🙂